src/services/Store/store/account.slice.ts

```diff
@@ -256,18 +279,8 @@ export const addTxToAccount = createAction<{
 export const startTxPolling = createAction(`${slice.name}/startTxPolling`);
 export const stopTxPolling = createAction(`${slice.name}/stopTxPolling`);
 
-// Polling Config
-const payload: IPollingPayload = {
-  startAction: startTxPolling,
-  stopAction: stopTxPolling,
-  params: {
-    interval: 5000,
-    retryOnFailure: true,
-    retries: 3,
-    retryAfter: 3000
-  },
-  saga: pendingTxPolling
-};
+export const startBalancesPolling = createAction(`${slice.name}/startPolling`);
```

Comment:
Please name this balance polling since we have two very similar actions in this slice.